upgrade/upgrade-instructions-for-improved-data-fixtures-and-tests.md

```diff
@@ -0,0 +1,63 @@
+- Modify your acceptance tests:
+    - Add two new modules - [`LocalizationHelper`](https://github.com/shopsys/shopsys/blob/v8.1.0/project-base/tests/ShopBundle/Test/Codeception/Helper/LocalizationHelper.php) and [`NumberFormatHelper`](https://github.com/shopsys/shopsys/blob/v8.1.0/project-base/tests/ShopBundle/Test/Codeception/Helper/NumberFormatHelper.php)
+        - The modules add new methods to acceptance tester that are useful in the tests, e.g.:
+            - `amOnLocalizedRoute()`
+            - `seeTranslationAdmin()`
+            - `seeTranslationFrontend()`
+            - `getFormattedPriceWithCurrencySymbolOnFrontend()`
+        - Rewrite your tests to make them resistant against domains and locales settings using these methods.
+        - Register the new modules in your `acceptance.suite.yml` configuration in `modules.enabled` section.
+    - Update your [`Cart/index.html.twig`](https://github.com/shopsys/shopsys/blob/v8.1.0/project-base/src/Shopsys/ShopBundle/Resources/views/Front/Content/Cart/index.html.twig) - add JS class `js-cart-item-price` it is used for identification of item price in cart in acceptance tests.
+- Do not forget to update your [`services.yml`](https://github.com/shopsys/shopsys/blob/v8.1.0/project-base/src/Shopsys/ShopBundle/Resources/config/services.yml) and [`services_test.yml`](https://github.com/shopsys/shopsys/blob/v8.1.0/project-base/src/Shopsys/ShopBundle/Resources/config/services_test.yml).
+- Run `php phing translations-dump` and fill in new translation for your locale in `dataFixtures.xx.po` (`xx` stands for your locale).
+- Update your [`easy-coding-standard.yml`](https://github.com/shopsys/shopsys/blob/v8.1.0/project-base/easy-coding-standard.yml) (some of the data fixture classes need to be excluded from some checks).
+- If you have extended any of the following classes, you need to update the constructor in your child as there are new dependencies:
+    - [`PriceExtension`](https://github.com/shopsys/shopsys/blob/v8.1.0/packages/framework/src/Twig/PriceExtension.php) now depends on [`CurrencyFormatterFactory`](https://github.com/shopsys/shopsys/blob/v8.1.0/packages/framework/src/Component/CurrencyFormatter/CurrencyFormatterFactory.php)
+    - [`LocalizationListener`](https://github.com/shopsys/shopsys/blob/v8.1.0/packages/framework/src/Model/Localization/LocalizationListener.php) now depends on [`AdministrationFacade`](https://github.com/shopsys/shopsys/blob/v8.1.0/packages/framework/src/Model/Administration/AdministrationFacade.php)
+    - [`NumberFormatterExtension`](https://github.com/shopsys/shopsys/blob/v8.1.0/packages/framework/src/Twig/NumberFormatterExtension.php) now depends on [`AdministrationFacade`](https://github.com/shopsys/shopsys/blob/v8.1.0/packages/framework/src/Model/Administration/AdministrationFacade.php)
+- Stop using `Domain::getAllIdsExcludingFirstDomain()` - the method will be removed in the next major release.
```

Comment:
```suggestion
- Stop using `Domain::getAllIdsExcludingFirstDomain()` - the method is deprecated and will be removed in the next major release.
```

Reply:
fixed in 3e53ddd53b6421b5025579dee4a29f0dcefefa85